sinks/sinks.go

```diff
@@ -35,6 +41,7 @@ const EventReportedCount = "sink.events_reported_total"
 // it's aggregated metrics.
 type MetricSink interface {
 	Name() string
+	Kind() string
```

Comment:
We have access to the sink kind already as `sinkConfig.Kind`, and can pass it in here:
https://github.com/stripe/veneur/blob/master/server.go#L425; do we want to either
- use this instead of hard-coding the value, or
- assert that the two are the same?

Reply:
I don't think it's plausible for us to use `sinkConfig.Kind` - we want to assume that every `MetricSink` implements `Kind()` because it's used within flusher.go. The assertion would be a nice touch but we can do it in a follow-up PR.